Fruit update

Rowtree opened the pull request:
:cl: Rowtree
rscadd: Adds 16 new fruits, and drinks + cocktails for them.
/:cl:

Adds 16 new fruits. 9 of which are alien drinks. 3 are Unathi, 3 are Skrell, and 3 are human alien fruits.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

Commit:
Fruit Expansion - Adds New Fruits and Recipes, adds 9 alien plants (3 unathi, 3 skrell, 3 human)

Commit:
Fruit Expansion - Adds New Fruits and Recipes, adds 9 alien plants (3 unathi, 3 skrell, 3 human)

Commit:
diona patch and spelling

Commit:
Fixes slices across all fruits

Commit:
removes whitespace

Commit:
Merge branch 'FruitUpdate' of https://github.com/Rowtree/Baystation12 into FruitUpdate

Spookerton commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm:
```diff
@@ -2813,3 +2813,151 @@
+	strength = 50
+
+	glass_name = "iridast berry juice"
+	glass_desc = "A intoxicating juice made from the Iridast Berry, common across human worlds."
```
An

Spookerton commented on code/modules/reagents/Chemistry-Recipes.dm:
```diff
@@ -2735,4 +2735,37 @@
+	result = /datum/reagent/ethanol/horchata
+	required_reagents = list(/datum/reagent/ethanol/rum = 2, /datum/reagent/drink/milk/cream = 1, /datum/reagent/cinnamon = 1, /datum/reagent/drink/syrup_vanilla = 1)
+	result_amount = 5
+	mix_message = "The mixture froths up into a rich, cool mixture"
```
opt: avoid repetition? rich, cool **liquid** maybe.

Spookerton commented:
@Rowtree also re-squash afterwards pleez.

Commit:
Various spelling fixes, makes Qokkrona lethal to humans.

Commit:
Fixes Qokkloa damage to a more appropriate number

Commit:
hotfixes

Commit:
Hotfixes

Commit:
parent 641d9425ffe8a467adb075445430512f9708c436
author Rowtree <DrRowtee@gmail.com> 1587315706 +0100
committer Rowtree <DrRowtee@gmail.com> 1587315723 +0100

Fruit Expansion - Adds New Fruits and Recipes, adds 9 alien plants (3 unathi, 3 skrell, 3 human)

Commit:
Merge branch 'FruitUpdate' of https://github.com/Rowtree/Baystation12 into FruitUpdate
